garden/src/pages/PipelinePage.tsx

```diff
@@ -6,6 +6,7 @@ import RelatedGardenBox from "../components/RelatedGardenBox";
 import DatasetBoxPipeline from "../components/DatasetBoxPipeline";
 import { searchGardenIndex } from "../globusHelpers";
 import Breadcrumbs from "../components/Breadcrumbs";
+import { NotebookViewer } from "../components/NotebookViewer";
```

Comment:
TODO: the default notebook styling with this off the shelf component doesn't match our color scheme

<img width="1671" alt="Screenshot 2023-11-03 at 4 46 16 PM" src="https://github.com/Garden-AI/garden-frontend/assets/6283143/8ef6ca8f-3945-49a2-ac95-bbd1dd71c7ef">

Reply:
This will be a good follow-on task for an intern